rllib/agents/cql/cql_tf_policy.py

```diff
@@ -46,30 +47,34 @@
 MEAN_MAX = 9.0
 
 
+def _repeat_tensor(t, n):
+    t_rep = tf.expand_dims(t, 1)
```

Comment:
can you add comments to explain the tensor maneuver here? otherwise the logics may be hard to follow.

Reply:
I added some comments but actually I just extended the existing code to work with an arbitrary number of dimensions. Not sure why the repetition process is like it is.